File: FizzBuzz/src/main/java/com/kata/FizzBuzz.java
package com.kata;

public class FizzBuzz {

    public static final int MIN_NUMBER = 1;
    public static final int MAX_NUMBER = 100;

public String getFizzBuzz (int number){
    if (number % 15 == 0) return "FizzBuzz";
    if (number % 3 == 0) return "Fizz";
    if (number % 5 == 0) return "Buzz";

    return String.valueOf(number);
};

public static void main(String[] args) {
       FizzBuzz fizzBuzz = new FizzBuzz();
       for (int i = MIN_NUMBER; i <= MAX_NUMBER; i++){
           System.out.println(fizzBuzz.getFizzBuzz(i));
        }
    }
}

File: 01-fizzbuzz/fizzbuzz-js/fizzbuzz.js
function fizzbuzz(number) {
  if (typeof number !== "number") throw new Error("parameter must be a number");

  if (number % 15 === 0) return "FizzBuzz";
  if (number % 3 === 0) return "Fizz";
  if (number % 5 === 0) return "Buzz";

  return number;
}

module.exports = { fizzbuzz };

// Permite ejecutarlo como script
if (require.main === module) {
  for (let i = 1; i <= 100; i++) {
    console.log(fizzbuzz(i));
  }
}

//npm start